### ISSUE TITLE
Varia: Heading inside Columns block on the homepage (using Shawburn theme) skips heading hierarchy

### ISSUE THREAD

**Issue by retnonindya** [Automattic/themes#2340]
## Steps to replicate

Create a page using Shawburn's homepage template/activate Shawburn theme on a website; then use accessibility tool to inspect the element:
https://wave.webaim.org/

## Result

Some headings are skipped (from H1, it jumps directly to H4):

![image](https://cldup.com/NUVWyZIQXT.png)
*Direct link: https://cldup.com/NUVWyZIQXT.png*

## Expected
The headings are using H2, H3, and so on.

The user also expressed their feedback for WordPress/Automattic to add more governance around theme design, specifically on approval and release -- if it doesn't meet accessibility standards, it needs to be improved.

* User report: 22325419-hc
